doc/src/explore/sui-explorer.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using the Sui Explorer
+---
+
+# Using Sui Explorer
+The [Sui Explorer](https://explorer.devnet.sui.io/) lets you view data about transactions and activity on the Sui network. In addition to viewing activity on the network, you can use the Explorer to:
+ * View up-to-date information about the activity and metrics on the Sui network.
+ * Look up, verify, and track your assets and contracts.
+ * Utilize fast, reliable, and transparent debugging and auditing data to help identify and resolve issues. 
+ * Get go-to-definition support for all smart contracts, referred to as packages in Sui.
+ * View validators and geographic locations of full nodes that are currently active.
+
+## Supported Browsers
+The Sui Explorer supports the following browsers:
+ * Chrome
+ * Firefox
+ * Safari
+ * Edge
+
+## Choose a network
+When you start Sui Explorer, it displays the transactions for the Devnet network by default. You can also use the Explorer to view data for a local network running in your environment or a custom RPC endpoint URL.
+
+
+## Finding your transaction
+You can search for the transactions using an address, object ID, or transaction ID. For example, you can search for your wallet address to confirm a transaction or view additional details about a transaction you’ve approved. See [Sui Wallet](../explore/wallet-browser.md) to learn how to create a wallet).  
+
+**To search for a transaction made using the Sui wallet**
+1. Open your Sui wallet.
+1. Click the clipboard icon to copy your wallet address.
+1. Open the Sui Explorer. Select the **Coins** tab if it is not currently selected.
+1. In the search field, paste your wallet address and then press **Enter**.
+
+The Explorer displays the **Address** details page for your wallet address. You should see the same transaction in Explorer that you see in your wallet history. Click on a transaction to view the details for it.
+
+## Explorer start page
+When you open the Sui Explorer, the page displays the transactions and top validators on the network. The **Transactions** table lists the transactions on the network with the most recent transaction first.
+
+The **Top Validators** table lists the top validators on the network and includes a map showing the geographic locations of all nodes on the network.
+
+Click **More Transactions** to open the **Transactions** page and view all of the transactions on the network.
+
+## Transactions page
+The **Transactions** page lists all transactions on the network. You can display 20, 40, or 60 rows of transactions per page. Use the drop-down near the bottom-right corner of the page to change the number of rows displayed per page. Use the page selector icons at the bottom of the page to view more transactions.
+
+### Transaction table fields
```

Comment:
One note is that these fields will change pretty soon when we change the transactions UI to be transfer-based, but I can update the docs when we do that.